cypress_test/Makefile.am

```diff
@@ -154,6 +154,7 @@ define start_coolwsd
 			--o:logging.file[@enable]=true --o:logging.level=trace \
 			--o:welcome.enable=false \
 			--o:user_interface.mode=$(USER_INTERFACE) \
+	        --o:accessibility.enable=true \
```

Comment:
Wrong indentation.

Reply:
Ah! In the file are used tabs in place of spaces. That's unusual. Anyhow, I fixed it.